src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenAGenerateRuntimeConfigurationFiles.cs

```diff
@@ -39,7 +40,7 @@ public void ItCanGenerateWithoutAssetFile()
             var task = new TestableGenerateRuntimeConfigurationFiles
             {
                 BuildEngine = new MockNeverCacheBuildEngine4(),
-                TargetFrameworkMoniker = ".NETCoreApp,Version=v3.0",
+                TargetFrameworkMoniker = $".NETCoreApp,Version=v{ToolsetInfo.CurrentTargetFrameworkVersion}",
                 RuntimeConfigPath = _runtimeConfigPath,
                 RuntimeConfigDevPath = _runtimeConfigDevPath,
                 RuntimeFrameworks = new[]
```

Comment:
I think the 3.0.0-preview1.100 should be a 6.0 version since it appears to be trying to roll forward on the latest minor if I'm reading this right.